File: gcp-natural-language-api-application/sentiments/analyze.py
from google.cloud import language
from google.cloud.language import enums
from google.cloud.language import types
import six

client = language.LanguageServiceClient()

content = 'This car is really pretty.'
content2 = 'This car is nice. It also gets terrible gas mileage!'

if isinstance(content2, six.binary_type):
    content2 = content2.decode('utf-8')

type_ = enums.Document.Type.PLAIN_TEXT
document = {'type': type_, 'content': content2}

response = client.analyze_sentiment(document)
sentiment = response.document_sentiment
print('Score: {}'.format(sentiment.score))
print('Magnitude: {}'.format(sentiment.magnitude))

File: gcp-natural-language-api-application/sentiments/analyze.js
async function analyze() {
    // Imports the Google Cloud client library
    const language = require('@google-cloud/language');

    // Instantiates a client
    const client = new language.LanguageServiceClient();

    // The text to analyze
    const text = 'This car is really pretty.';
    const text2 = 'This car is nice. It also gets terrible gas mileage!';

    const document = {
        content: text2,
        type: 'PLAIN_TEXT'
    };

    // Detects the sentiment of the text
    const [result] = await client.analyzeSentiment({document: document});
    const sentiment = result.documentSentiment;

    console.log(`Text: ${text}`);
    console.log(`Sentiment score: ${sentiment.score}`);
    console.log(`Sentiment magnitude: ${sentiment.magnitude}`);
}

analyze().catch(console.error);
